### DISCUSSION TITLE
whisper with googlecolab have some error ,,, when i click summit file  on the menu then the error happen

### DISCUSSION THREAD

**Discussion by maloadjav** [openai/whisper#2340] [Q&A]
i use whisper on google here 

https://colab.research.google.com/drive/1qeTSvi7Bt_5RMm88ipW4fkcsMOKlDDss?usp=sharing#scrollTo=IuL5nGTIWYTY

i just click and let the program run... 

(1 error ) warning form code windows

---------------------------------------------------------------------------------------------------------------------------------------------------
!cd whisper-webui/ && python app.py --input_audio_max_duration -1 --share True

2024-09-13 11:57:01.853476: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-09-13 11:57:01.873480: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-09-13 11:57:01.879328: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-09-13 11:57:01.893264: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-09-13 11:57:03.095696: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Using whisper implementation: whisper
IMPORTANT: You are using gradio version 3.38.0, however version 4.29.0 is available, please upgrade.
IMPORTANT: You are using gradio version 3.38.0, however version 4.29.0 is available, please upgrade.
IMPORTANT: You are using gradio version 3.38.0, however version 4.29.0 is available, please upgrade.
Queue mode enabled (concurrency count: 1)
IMPORTANT: You are using gradio version 3.38.0, however version 4.29.0 is available, please upgrade.
Running on local URL: http://127.0.0.1:7860/
Running on public URL: https://487be5378fc94e8a7d.gradio.live/

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run gradio deploy from Terminal to deploy to Spaces

-------------------------------------------------------------------------------------------------------------------------------------------------

anyway i see lots of this messages ( plx upgrade gradu ) in the past but i can run the program until the end
but today i cant do it

(2)

i open the menu that gradu create it

i upload file , click submit and i got error

i back to the code window and the program report something like this
-----------------------------------------------------------------------------------------------------------------------------------
Creating whisper container for whisper
Using cache found in /root/.cache/torch/hub/snakers4_silero-vad_master
Created Silerio model
Getting timestamps from audio file: /tmp/gradio/12b56556bae9cd22583a6bee121d42856438858f/7v.mp4, start: 0, duration: 604.071
Processing VAD in chunk from 00:00.000 to 10:04.071
VAD processing took 11.801706895999814 seconds
Transcribing non-speech:
[{'end': 6.758, 'start': 0.0},
{'end': 519.898, 'gap': True, 'start': 6.758},
{'end': 531.61, 'expand_amount': 7.988000000000056, 'start': 519.898},
{'end': 538.886, 'start': 531.61},
{'end': 604.071, 'gap': True, 'start': 538.886}]
Processing timestamps:
[{'end': 6.758, 'start': 0.0},
{'end': 519.898, 'gap': True, 'start': 6.758},
{'end': 531.61, 'expand_amount': 7.988000000000056, 'start': 519.898},
{'end': 538.886, 'start': 531.61},
{'end': 604.071, 'gap': True, 'start': 538.886}]
Running whisper from 00:00.000 to 00:06.758 , duration: 6.758 expanded: 0 prompt: None language: None
Loading whisper model medium
Whisper took 18.0588212939997 seconds
Running whisper from 00:06.758 to 08:39.898 , duration: 513.14 expanded: 0 prompt: でもちょっと なんであげるの language: japanese
Whisper took 69.36960414399982 seconds
Running whisper from 08:39.898 to 08:51.610 , duration: 11.711999999999989 expanded: 7.988000000000056 prompt: None language: japanese
Whisper took 1.5131064330003028 seconds
Running whisper from 08:51.610 to 08:58.886 , duration: 7.275999999999954 expanded: 0 prompt: None language: japanese
Whisper took 2.8751684230001047 seconds
Running whisper from 08:58.886 to 10:04.071 , duration: 65.18500000000006 expanded: 0 prompt: None language: japanese
Whisper took 5.790672107000319 seconds
Created JSON file /tmp/tmpegzful_3/7vmp4-result.json
Max line width 40
Deleting source file /tmp/gradio/12b56556bae9cd22583a6bee121d42856438858f/7v.mp4
-----------------------------------------------------------------------------------------------------------------------------
(3)

seem like the program run and create 1 line of subtitle before its delete the file that i uploaded

i back to grado menu and click flag

the program saving something

then i back to code windows and i see the program report this

Error while flagging: A FileSerializable component can only deserialize a string or a dict, not a <class 'bool'>: False

how can i fixed this ??

**Comment by phineas-pta** [comment]
u should ask the author of that code, this has nothing to do with whisper

**Reply by maloadjav** [reply]
okay.thank you so mcuh
